pkgs/applications/misc/kaufkauflist/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildNpmPackage, fetchFromGitea, nix-update-script }:
+
+buildNpmPackage rec {
+  pname = "kaufkauflist";
+  version = "1.0.0";
+
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = "annaaurora";
+    repo = "kaufkauflist";
+    rev = "v${version}";
+    sha256 = "sha256-VuRokNL+/MG/bO7JpqE0TjoWHhCv13oYGtoODMkceNY=";
+  };
+
+  npmDepsHash = "sha256-cEkInUAniBwOH4//MisffXest9KAmT0M5dnAdGcQ9+o=";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cp -vr build/* $out/
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
uh, that's unfortunate

Reply:
> 2. You should expose the PocketBase schema somewhere, I'd say.

should I do it like before?: have `website` directory and `pb_schema.json` in the result?

Reply:
"like before"? I didn't go through all of the past revisions of this PR, is that what you mean? (If so, why did you remove it?)

And the structure really doesn't matter, we (sadly) don't have much convention. Though I would say it's a good idea to put at least the web files in `$out/share/kaufkauflist`, not sure where the best place for the schema would be though, since I doubt you want that being served.

Reply:
> nixpkgs-update doesn't work with Gitea at the moment.

should I remove it or leave it in so it'll work when nixpkgs-update does support Gitea?

Reply:
Remove it, it'll fail and there's no reason to add a script that'll fail :D

Feel free to add a TODO to add it, though!

Reply:
I commented it with a todo.

Reply:
And put stuff in `$out/share/kaufkauflist` and `$out/share/pocketbase` like you suggested.